src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Component.razor

```diff
@@ -298,15 +298,18 @@
 <div class="rating-container">
     <h2>Rating</h2>
     <h5>Large stars</h5>
-    <BitRating DefaultRating="5" Size="RatingSize.Large" />
+    <BitRating Size="RatingSize.Large" />
     <h5>Small stars, with 0 stars allowed</h5>
     <BitRating AllowZeroStars="true" />
     <h5>10 small stars</h5>
-    <BitRating Max="10" />
+    <div>
+        <BitRating Max="10" OnChange="@(arg => OnRatingChange(arg))" />
+        <span>@RatingValue stars selected</span>
+    </div>
     <h5>Disabled</h5>
-    <BitRating DefaultRating="5" IsEnabled="false" />
+    <BitRating IsEnabled="false" />
     <h5>ReadOnly</h5>
-    <BitRating DefaultRating="5" IsReadonly="true" />
+    <BitRating IsReadonly="true" />
     <h5>Custom icons</h5>
     <BitRating Icon="HeartFill" UnselectedIcon="Heart" />
```

Comment:
Please provide a sample in which the value is changing from outside of the component.
check the BitCheckBox samples, please.